.vscode/cspell.json

```diff
@@ -26,7 +26,6 @@
     "sdk/**/_generated/**",
     "**/*requirement*.txt",
 
-    "sdk/anomalydetector/azure-ai-anomalydetector/**",
```

Comment:
This is not the SB SDK, so we don't want to touch this. This should be reverted.